public/css/presenter.css

```diff
@@ -118,6 +118,15 @@
       width: 100px;
     }
 
+    #timerDisplay.paused {
+      filter: grayscale(100%);
```

Comment:
Needs -webkit prefix to work with Chrome, Safari, and Opera.  Not supported by any version of IE.